hbase-protocol-shaded/src/main/protobuf/ClusterStatus.proto

```diff
@@ -146,6 +146,9 @@ message RegionLoad {
 
   /** the current total coprocessor requests made to region */
   optional uint64 cp_requests_count = 20;
+
+  /** the number of references active on the store */
+  optional int32 store_ref_count = 21 [default = 0];
```

Comment:
uint32?

Reply:
Reference count may go negative. It would be a bad bug. We want to see it. I know you can guess if a really large value means a negative integer was cast to unsigned but it should be explicit if it goes negative.

Reply:
Also the int vs long. Don't really care but think refcount > INT_MAX is going to be a bug (smile)